src/ontology/mondo-edit.obo

```diff
@@ -570175,6 +570185,3049 @@ intersection_of: MONDO:0700097 MONDO:0000721 ! cross-species analog xanthinuria
+name: squamous cell carcinoma of the digit, non-human animal
+xref: OMIA:001787 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001787"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:001787", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011823
+name: hemangiosarcoma, non-human animal
+xref: OMIA:001813 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001813"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:001813", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011824
+name: horn cancer, non-human animal
+xref: OMIA:001859 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001859"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:001859", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011825
+name: choroid plexus tumour, non-human animal
+xref: OMIA:002403 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002403"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:002403", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011826
+name: dysplastic gangliocytoma of the cerebellum, non-human animal
+xref: OMIA:002529 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002529"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:002529", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011827
+name: congenital mast cell tumor, non-human animal
+xref: OMIA:002578 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002578"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:002578", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011828
+name: cancer, non-human animal
+xref: OMIA:002587 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002587"} ! non-human animal disease
+is_a: MONDO:1011305 {source="OMIA:002587", source="https://orcid.org/0000-0002-5002-8648"} ! cancer or benign tumor, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
```

Comment:
doesn't this one have a human x-species analog?

Reply:
yes, added